### DISCUSSION TITLE
Show alt-text for images where available

### DISCUSSION THREAD

**Discussion by hirnsushi** [jocmp/capyreader#388] [Feature Requests]
It would be great if the alt-text of images could be displayed underneath (in light gray perhaps), or maybe in the fullscreen image viewer.

My personal usecase is the [xkcd feed](https://xkcd.com/rss.xml) where part of each comic is the captions in the alt-text.
It might also benefit accessibility, although I'm not a screen reader user myself so I can't speek to how the app is presented in that context, it might already read out alt-text. At the very least it would make it possible to get to the additional information the author might have added.

**Comment by jocmp** [comment]
hey @hirnsushi - I haven't forgotten about this. I think it's a good idea and I like the suggestion to show it as a description in the fullscreen image. I'm tracking this here: https://github.com/jocmp/capyreader/issues/427

**Reply by jocmp** [reply]
Small update. I have alt text working in the fullscreen viewer. I'll include it in the next release. Here's a demo of it working with xkcd.

<img src="https://github.com/user-attachments/assets/769ac67a-2ae5-49d2-9670-1a56fcb86479" width="400px">

**Reply by jocmp** [reply]
This feature is live in the latest build, [2024.11.1075-dev](https://github.com/jocmp/capyreader/releases/tag/2024.11.1075-dev).
